baselines/teams.md

```diff
@@ -88,114 +87,141 @@ need.
-- _Rationale:_ TODO add rationale.
-- _Last modified:_ July 2023
+4.  Under the **Meeting join & lobby** section **,** ensure
+    **Who can bypass the lobby** is set to **People in my org, trusted orgs, and guests**.
 
-#### MS.TEAMS.3.3v1
-B2B guest users MAY be admitted automatically.
-- _Rationale:_ TODO add rationale.
-- _Last modified:_ July 2023
+5.  In the same section, set **People dialing in can bypass the lobby** to
+    **Off**.
 
-#### MS.TEAMS.3.4v1
-MS.TEAMS.3.1.1, MS.TEAMS.3.2.1, and MS.TEAMS.3.3.1 SHOULD be applied in the Global (Org-wide default) meeting policy.
-- _Rationale:_ TODO add rationale.
-- _Last modified:_ July 2023
 
-#### MS.TEAMS.3.5v1
-Custom meeting policies MAY be created that allow more flexibility for specific users.
-- _Rationale:_ TODO add rationale.
+## 3. Starting Teams Meetings
+
+This setting controls which meeting participants can start a meeting. In
+this instance, the term “anonymous users” refers to any Teams users
+joining calls that are not authenticated through the agency’s tenant.
+
+### Policies
+
+#### MS.TEAMS.3.1v1
+Anonymous users SHALL NOT be enabled to start meetings in the Global (Org-wide default) meeting policy or in custom meeting policies if any exist.
+- _Rationale:_ For any agencies that implemented custom policies that provided more flexibility to some users to automatically admit "everyone" to a meeting - this policy provides protection from anonymous users starting meeting to scrape internal contacts. 
 - _Last modified:_ July 2023
 
 ### Resources
 - [Meeting policy settings - Participants & guests \| Microsoft
-  Docs](https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-participants-and-guests)
+Docs](https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-participants-and-guests)
 
 ### License Requirements
 - N/A
 
 ### Implementation
-To configure settings for automatic meeting admittance:
+All the settings in this section are configured in the [**Microsoft Teams admin center**](https://admin.teams.microsoft.com). 
+#### MS.TEAMS.3.1v1, instructions:
+To configure settings for anonymous users:
+
 
 1.  Sign in to the [**Microsoft Teams admin
     center**](https://admin.teams.microsoft.com).
-
-2.  Select **Meetings -\>** **Meeting** **policies**.
+2.  Select **Meetings >** **Meeting policies**.
 
 3.  Select the **Global (Org-wide default)** policy.
 
-4.  Under the **Participants & guests** section**,** ensure
-    **Automatically admit people** is *not* set to **Everyone**.
-
-5.  In the same section, set **Dial-in users can bypass the lobby** to
-    **Off**.
+4.  Under the **Meeting join & lobby** section **,** set **Anonymous users and dial-in callers can start a meeting** to **Off.**
 
-6.  If custom policies have been created, repeat these steps for each
+5.  If custom policies have been created, repeat these steps for each
     policy, selecting the appropriate policy in step 3.
 
 ## 4. External User Access
```

Comment:
The hyperlink to the OMB Max Site is presumably there to provide a list of .gov domains but what is a user supposed to do with this information?

- The OMB Max Site requires a login. Are all government users supposed to be able to login there? Once they are in, what are they supposed to do? We don't provide any instructions for navigating that site. 
- Are we expecting a user to enter the list of every government .gov domain into the Teams allowable domain configuration?